syntax = "proto3";

message Token {
    
}

message LoginResult {
    oneof result {
        string error = 1;
        Token token = 2;
    }
}
